shared/js/media/media_frame.js

```diff
@@ -47,6 +47,8 @@ function MediaFrame(container, includeVideo, maxImageSize) {
   this.image = new Image();
   this.image.className = 'image-view';
   this.image.style.opacity = 0;                        // Start off hidden.
+  this.image.style.transitionProperty = 'opacity';
+  this.image.style.transitionDuration = '100ms';
```

Comment:
Add transition property and duration for smooth transition effect when opacity changes